sample/CosmosDBTrigger-CSharp/run.csx

```diff
@@ -0,0 +1,9 @@
+﻿#r "Microsoft.Azure.Documents.Client"
+using Microsoft.Azure.Documents;
+using System.Collections.Generic;
+using System;
+public static void Run(IReadOnlyList<Document> input)
+{
+    Console.WriteLine("Documents modified " + input.Count);
```

Comment:
Using `Console.WriteLine` doesn't work from a function -- add a parameter `ILogger logger` and do a `logger.LogInformation("")` instead. That'll send the log to the file logs, App Inisights, etc.